pkg/support/support.go

```diff
@@ -196,12 +214,50 @@ func (bb *bundleBuilder) copyInFile(writer *zip.Writer, fileName string, prefix
 		return err
 	}
 
-	_, err = zipCopy.Write(redacted)
+	buf := new(bytes.Buffer)
+	_, err = buf.ReadFrom(f)
 	if err != nil {
 		return err
 	}
 
-	return nil
+	return bb.copyAndRedactFile(zipCopy, buf)
+}
+
+func (bb *bundleBuilder) streamFileFromVM(writer *zip.Writer, filename, prefix string) error {
+	pipeReader, pipeWriter := io.Pipe()
+	errBuf := new(bytes.Buffer)
+
+	_, filePathInVM, _ := strings.Cut(filename, ":")
+	cmd := bb.lcc.CreateWithoutStdio("shell", "finch", "sudo", "cat", filePathInVM)
+	cmd.SetStdout(pipeWriter)
+	cmd.SetStderr(errBuf)
+
+	err := cmd.Start()
+	if err != nil {
+		return err
+	}
+
+	waitStatus := make(chan error)
+	go func() {
+		err := cmd.Wait()
+		_ = pipeWriter.Close()
+		waitStatus <- err
```

Comment:
Does the error returned from `cmd.Wait()` display the correct error message, like "file not found"? Maybe we should read `errBuf` to generate the error message.